.github/workflows/codecoverage.yml

```diff
@@ -0,0 +1,60 @@
+name: Code Coverage (Apollo tests)
+on: [push, pull_request]
+# on:
+#   schedule:
+#     - cron: "0 3 * * 0" # Every Sunday at 3AM UTC
```

Comment:
Just make sure to un-comment this.

Reply:
Yeah let me revert those changes and merge this PR.
Thanks